litellm/proxy/guardrails/guardrail_hooks/acuvity.py

```diff
@@ -0,0 +1,89 @@
+import os
+from typing import Any, Dict, List, Literal, Optional, Union
+
+import litellm
+from acuvity import Acuvity, Security
+from litellm._logging import verbose_proxy_logger
+from litellm.caching.caching import DualCache
+from litellm.integrations.custom_guardrail import CustomGuardrail
+from litellm.litellm_core_utils.logging_utils import (
+    convert_litellm_response_object_to_str,
+)
+from litellm.proxy._types import UserAPIKeyAuth
+from litellm.proxy.common_utils.callback_utils import (
+    add_guardrail_to_applied_guardrails_header,
+)
+from litellm.proxy.guardrails.guardrail_helpers import should_proceed_based_on_metadata
+from litellm.types.guardrails import GuardrailEventHooks
+
+GUARDRAIL_NAME = "acuvity"
+
+class AcuvityGuardrail(CustomGuardrail):
+    def __init__(
+        self, api_key: Optional[str] = None, api_base: Optional[str] = None,
+        **kwargs,
+    ):
+        # store kwargs as optional_params
+        self.optional_params = kwargs
```

Comment:
Extract our configuration right here, and try to validate the guard configuration here. If the validation of the config fails, this should raise an exception and fail everything (ideally). Broken configuration must fail initialization.